services/galley/schema/src/V55_TeamFeatureSelfDeletingMessagesPaymentStatus.hs

```diff
@@ -0,0 +1,34 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2020 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V55_TeamFeatureSelfDeletingMessagesPaymentStatus
+  ( migration,
+  )
+where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration = Migration 55 "Add payment status config for self-deleting messages" $ do
+  -- 0: Payment Status locked, 1: Payment Status unlocked
```

Comment:
This is clear from the Cql instance, and a comment here is at risk of getting outdated.  Just remove?  Or link the haskell type or its Cql instance?

Reply:
Done